### DISCUSSION TITLE
Uso da APyB pela PyLadies Brasil

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#16] [Geral]
[Mensagem original enviada para a lista de e-mail da APyB em 16 dez. 2022]

Bom dia!!

Felicitamos a nova gestão da APyB!!
Celebramos muito que 50% da Diretoria seja composta por PyLadies. Acreditamos que isso fortalecerá nossa comunidade Python Brasil. ✨

Na gestão anterior demonstramos interesse em ter apoio da APyB no tema financeiro contábil, ou seja, gostaríamos de usar a Associação como pessoa jurídica para a PyLadies Brasil. Então, gostaríamos de retomar o diálogo com a nova gestão.

Atualmente, nosso caixa encontra-se na conta de uma pessoa física (@jtemporal) . Por isso, gostaríamos de orientações sobre como realizar a transferência para a Associação, além disso acordar o processo para uso do dinheiro em caixa.

**Comment by rennerocha** [comment]
Vou colocar algumas mensagens que enviei nessa thread de e-mail citada, pois acho que pode ajudar na discussão:

> Enviar o dinheiro para a conta da APyB com certeza entrará como uma doação e não seria um problema.
> 
> Agora precisamos entender como seria a gestão desse valor, já que não existe uma subconta da PyLadies Brasil dentro da conta da APyB. Considerando a questão de transparência que vocês mencionaram (que a APyB já precisa ter também), a ideia é que esse valor continue "separado" dos outros gastos da APyB (como se houvesse um saldo da PyLadies Brasil que não pudesse ser utilizado para outras ações da APyB)?
> 
> Isso é um questionamento que já tivemos com saldos remanescentes de eventos e que não poderiam ser utilizados em outras edições justamente pelo dinheiro estar "misturado".

Das respostas dessa minha pergunta, ficou claro para mim que as PyLadies gostariam de poder manter esse dinheiro "separado" do dinheiro regular da APyB, por questão de transparência e facilidade para gestão dele.

Então minha opinião final:

> Realmente a PyLadies Brasil tem um funcionamento bem diferente de uma organização de um evento pontual. São vários eventos e iniciativas apoiadas de diferentes maneiras.
> 
> Dando uma opinião como parte do Conselho Fiscal (aí a diretoria decide o que é melhor), abrir uma nova conta bancária da APyB em outro banco para manter esses valores, seria a maneira mais fácil de manter o dinheiro separado e dar a transparência necessária para a PyLadies e para os associados da APyB. Assim não teríamos que manter nenhuma contabilidade separada para conseguir "separar" virtualmente o que é APyB e o que é PyLadies.
> 
> Agora como associado, eu acho muito legal se a APyB conseguir ajudar as PyLadies dessa maneira! É esse tipo de coisa que a Associação precisa sempre fazer!

**Comment by felipythondev** [comment]
A APyB não poderia simplesmente utilizar uma outra conta para isso? Exemplo, a conta principal da APyB é no Banco do Brasil e o dinheiro das Pyladies fica em uma conta também da APyB só que no Nubank(ou qualquer outro banco).

**Reply by gabubellon** [reply]
Ou até mesmo tem multiplas contas poupanças no BB para isso ? 
@muriloviana sabe  se conseguimos ter mais de 1 conta poupança pelo menos ?

**Reply by rennerocha** [reply]
@gabubellon não é possível ter mais de uma conta poupança em uma conta bancária. Por sinal, conta poupança para PJ não é algo tão interessante de se manter, pois paga-se imposto sobre o rendimento. Abrir uma conta em outro banco me parece a saída mais simples, menos burocrática e mais prática para todos os envolvidos, já que a separação do dinheiro APyB e o dinheiro PyLadies é "física" não tendo a menor possibilidade de eles serem misturados.

**Reply by muriloviana** [reply]
Respondendo as perguntas:

* @lipemorais a ideia de abrir uma nova conta bancária é uma das opções que estamos levando em consideração. O único porém, é que como a APyB é uma associação privada sem fins lucrativos (seus associados não recebem nem um centavo de possíveis lucros) são poucos os bancos que permitem/tem interesse na abertura de conta para tais entidades, o Nubank, por exemplo, não tem. Estamos vendo outras opções, e sugestões são sempre bem-vidas \o/
* @gabubellon infelizmente não é possível ter mais de uma conta poupança por conta, e não sei se seria a melhor opção por causa da desvalorização do montante.
* @rennerocha, sobre a poupança, hoje uma parte do dinheiro da associação está em um investimento financeiro no próprio Banco do Brasil: RF Ref DI Plus Ágil - CNPJ: 05.943.661/0001-74

**Reply by rennerocha** [reply]
@muriloviana 
Sobre contas bancárias para associações, praticamente nenhum banco digital oferece essa opção. Mesmo para empresas com sócio é difícil encontrar um banco digital que aceite (e eu procurei bastante, tanto para minha empresa, quando para o meu hackerspace). No final, a melhor opção é realmente um banco tradicional (minha empresa tem uma conta no Itau, e o LHC tem conta no Bradesco). A mensalidade em praticamente todos esses bancos é em torno de R$110 para o pacote mais básico, além de que (pelo menos nesses dois), o saldo em conta-corrente fica num daqueles investimentos automáticos rendendo diariamente. A Caixa tem uma mensalidade menor, mas eu fugiria deles de qualquer maneira, pois é mto difícil resolver qqer coisa não tão complexa sem ter que ir fisicamente na sua agência (tem certas coisas que tem que ser na sua agência mesmo!)

**Comment by tiidadavena** [comment]
A APyB está verificando com as PyLadies qual seria a melhor data e hora para marcar uma reunião a fim de compreender o estado atual e providenciar os próximos passos. 
É mister ressaltar que os pontos levantados até o momento e os próximos serão registrados aqui.

**Reply by tiidadavena** [reply]
@cecivieira poderias ver junto com as PyLadies qual o melhor horário e data para marcar a reunião?

**Reply by cecivieira** [reply]
Na terça-feira (08/03) pela manhã, sugeriremos 3 horários para a reunião.

**Comment by gabubellon** [comment]
@cecivieira uma dúvida existe hoje uma "hierarquia fomal" como existe na APyB na Pyladies Br/Mundo ? 

Outra ponto, gostartia de trazer uma sugestão aqui quanto a isso, pelo que entendi de conversar passadas a ideia é ter o dinheiro tutelado sobre a APyB uma vez que isso centraliza o mesmo dentro de uma pessoa juridica e se tem a segurança gerada por isso.
Isso pode ser um bom case para validar o sucesso da APyB centralizar de maneira separada para outra iniciativas de Python em cenários que queiram isso.

Minha proposta é levar para discussão e aprovação a criação de cargos dentro da APyB para as pessoas que iram auditar e tomar decisões sobre esse dinheiro. Uma "delegação" da PyLadies no caso, que seria responsável de gerir o mesmo junto com a APyB, uma vez que entendo que as decisões de uso desse dinheiro é só da PyLadies, mas precisa estar aderente as comunicações da APyB.

**Reply by cecivieira** [reply]
@gabubellon Na rede de comunidades PyLadies no Brasil não existem cargos. O que há são pessoas mais ativas e que acabam movimentando um pouco mais as atividades, acredito que será interessante ter "representantes" da rede junto a APyB para autorizar a movimentação do dinheiro.

**Reply by rennerocha** [reply]
@gabubellon existe uma questão prática que será necessário confirmar com o contador a respeito de quem é autorizado a movimentar essas contas bancárias. Por exemplo, no LHC (https://lhc.net.br), somente o Diretor Presidente e o Diretor Financeiro podem efetuar movimentações (com a aprovação dos dois apenas). Eu acredito que pelo estatuto da APyB, somente o presidente e o financeiro podem realizar essas movimentações em qualquer conta bancária, então isso continuaria centralizado. Não sei se é possível dar poderes plenos a outra pessoa (por procuração, não sei, por isso verificar com a contadora) de movimentar uma outra conta bancária. Também não sei se isso é de interesse da APyB.

**Reply by gabubellon** [reply]
@rennerocha eu não digo em movimentar o dinheiro as is, com acesso a conta. Mas ser a pessoa a tomar as decisões e auditar as movimentações pela APyB. Como uma CFO da PyLadiesBR dentro da APyB

**Reply by rennerocha** [reply]
@gabubellon a ideia de ter uma pessoa oficialmente designada da PyLadiesBR que possa tomar a decisão de autorizar ou não a movimentação de dinheiro do grupo é muito interessante. Por se tratar de dinheiro, e as vezes quantias não insignificantes, precisamos ter uma definição clara de quem pode ou não ter essa decisão. O ideal seria que isso fosse oficializado na associação. Ou com uma alteração em nosso estatuto, incluindo essa representante dentro da estrutura da APyB (não necessariamente diretoria), ou em nosso Regimento Interno (que pode ser alterado com mais facilidade).

Já que parece que estamos caminhando para consultar uma advogada em #14 , podemos nessa mesma consulta, pedir sugestões legais para tornar isso o mais transparente e seguro possível.

@cecivieira talvez meu comentário esteja pedindo formalismo demais, só que no momento as PyLadies envolvidas com certeza não dariam problemas, mas temos que estar prevenidos legalmente para que no futuro não se abra a possibilidade de termos conflitos por causa de dinheiro.

**Reply by cecivieira** [reply]
@rennerocha @gabubellon talvez não tenha ficado explícito no meu último comentário, mas também achamos relevante ter uma indicação oficial da parte das PyLadies para autorizar a movimentação de dinheiro junto a APyB. Essa(s) representante(s) pode(m) ser designada(s) nas nossas assembleias anuais.

Acredito que esses detalhes sobre viabilização da tutela do dinheiro, prestação de contas, etc começarão a ser alinhados na  primeira reunião.

**Reply by rennerocha** [reply]
@cecivieira sim, com certeza isso será alinhado na reunião. Só fiz o comentário de possibilidades para que tbm possam ser discutidos na reunião se acharem necessário.

**Comment by cecivieira** [comment]
Bom dia!

Sugerimos a realização da reunião no dia 14 de março (segunda-feira) as 17h.
Conforme combinado com @muriloviana , segue o link para a videochamada:

https://meet.google.com/yuo-fzbm-cqj

**Comment by tiidadavena** [comment]
@cecivieira, infelizmente o horário das 17h não permite a presença dos membros da Diretoria. 

Assim sendo, podemos marcar a reunião para às 19h na mesma data (14/03)?

**Reply by cecivieira** [reply]
Podemos sim. Reagendei o evento no calendário. O link permanece o mesmo

**Comment by cecivieira** [comment]
A ata da primeira reunião entre APyB e PyLadies Brasil:
https://github.com/pyladies-brazil/organizacao/issues/84

**Comment by cecivieira** [comment]
@anadulce @tiidadavena @rougeth @muriloviana Será possível a APyB entregar a ação "APyB: Documentar prosposta de utilização do sistema financeiro da APyB por parte das PyLadies, definir regras e fluxo. A ser feito antes da assembleia das PyLadies - prazo de entrega 29 de março." no prazo estabelecido?

**Reply by rougeth** [reply]
Sim :)

**Reply by anadulce** [reply]
# Proposta de utilização da infraestrutura financeira da APyB por parte das PyLadies

## Contextualização

Na última [Assembleia anual das PyLadies Brasil, realizada em 16 de outubro de 2021](https://github.com/pyladies-brazil/organizacao/issues/79), ficou determinado que as PyLadies solicitariam a possibilidade da utilização da infraestrutura financeira da APyB para manter seu caixa, que atualmente se encontra em responsabilidade de pessoas físicas pertencentes à comunidade.

A comunicação junto a APyB começou a ser realizada na lista de e-mail da associação em 16 dezembro de 2021, seguiu para a [Lista de Discussão](https://github.com/apyb/comunidade/discussions/16) e uma [primeira reunião foi realizada no dia 14 de março de 2022](https://github.com/pyladies-brazil/organizacao/issues/84) entre APyB e PyLadies para definir os direcionamentos.

## Referência da utilização da infraestrutura da APyB por eventos

Os critérios para usar a infraestrutura da APyB são simples:

- A organizadora e responsável pelo evento precisa ser associada da APyB.
- O evento deve ter relação com a missão da APyB que é fomentar Python no Brasil.
- O evento precisa ser sem fins lucrativos.

Para mais detalhes, segue o [documento](https://docs.google.com/document/d/1j0UwPbZrN1E8W3zdclzyIxaFPgztjLAATSdsPAizFX0/edit?usp=sharing) que dispõe na íntegra sobre a utilização da infraestrura da APyB por eventos para fins de referência.


## Proposta de utilização da infraestrutura financeira da APyB pelas PyLadies

A APyB dispõe às PyLadies um modelo análogo ao utilizado pelos eventos, como a Python Brasil: a utilização de sua conta principal no Banco do Brasil, com visualização de entradas e saídas disponibilizadas, e solicitação de entradas, pagamentos e reembolsos por meio dos processos já existentes. Tais registros serviriam para garantir lisura e controle das movimentações.

De acordo com o documento [Como organizar eventos usando a infraestrutura da APyB](https://docs.google.com/document/d/1j0UwPbZrN1E8W3zdclzyIxaFPgztjLAATSdsPAizFX0/), é importante ressaltar que o montante da conta virtual das PyLadies não pode ser utilizado em eventos os quais hajam cobrança de ingresso.

Condicionamos a aprovação da solicitação de uso da infraestrutura financeira da APyB pelas PyLadies à definição, por parte das PyLadies, de um modelo de autorização de transações. O objetivo de tais autorizações é determinar quais requisições de pagamentos enviadas à APyB em nome das PyLadies foram aprovadas por elas, pois consideramos que devido ao modelo descentralizado de organização das PyLadies, quaisquer PyLadies poderiam solicitar o uso de seus recursos, e que deverão passar por um processo de validação interno.

Para efetivar essa solicitação, a Associação ressaltou a necessidade de que o formulário (interligado com a planilha de acompanhamento financeiro) de utilização da infraestrutura financeira da APyB seja preenchido anualmente por integrantes das PyLadies, representando a Rede PyLadies Brasil.

Ressaltamos que as PyLadies que representarão a Rede PyLadies Brasil perante a APyB necessariamente precisam ser associadas efetivas com mensalidades em dia da APyB.

Todo o montante sob guarda da APyB estará disponível para as PyLadies por um período indeterminado, até o momento em que solicitarem o encerramento da utilização de recursos da APyB, e sua movimentação estará condicionada à estarem em dia com o preenchimento anual do formulário de utilização da infraestrutura financeira da APyB.

**Reply by anadulce** [reply]
Ressaltamos que estamos abertos à dúvidas e também a discutir essa proposta inicial.

**Reply by rennerocha** [reply]
De qual volume de dinheiro e quantidade de movimentações estamos falando?

Acho que não dá para tratar as PyLadies da mesma maneira que um evento. Um evento tem início, meio e fim e após o encerramento dele, os valores financeiros são (pelo menos no momento está funcionando dessa maneira) incorporados ao saldo geral da APyB. Estamos falando de um arranjo diferente com valores que podem ser usados em diversos eventos diferentes, com regras de liberação diferentes, envolvendo inclusive uma outra organização (não-formal, mas que tem suas regras de governança)

Na minha opinião, manter o dinheiro das PyLadies, na conta principal do Banco do Brasil, como está sendo proposto, vai aumentar o trabalho de se separar esses valores. Manter uma conta bancária separada, torna muito mais simples isso (qualquer movimentação ali será das PyLadies e pronto). O custo mensal de uma conta é de ~R$100 (Itau e Bradesco como referência). Acredito que os valores que serão movimentados e que são arrecadados pelas PyLadies sejam suficientes para manté-la com sobras.

**Reply by rennerocha** [reply]
> De acordo com o estatuto da APyB, é importante ressaltar que o montante da conta virtual das PyLadies não pode ser utilizado em eventos os quais hajam cobrança de ingresso.

De qual artigo do estatuto estamos falando? Não encontrei nenhum artigo proibindo isso.

**Reply by rougeth** [reply]
Eu não vejo tanta diferença em relação aos eventos Renne. Com a exceção de que o valor permanece (ou passa de um ano para outro - "evento" pyladies2022 -> pyladies2023), as regras são as mesmas já existentes. Consideraremos essa conta virtual das PyLadies como evento que não cobrará ingresso, ou seja, isento da taxa da APyB e, dessa forma, o dinheiro não poderá ser usado em eventos que cobram ingresso - suficiente para esse caso.

Concordo 100% que ter uma outra conta seria o cenário mais simples no sentido de manter o controle de entrada e saída, inclusive não estamos descartando essa ideia. Dito isso, acho que a nossa proposta é suficiente para pelo menos começarmos a suportar o caso delas e validarmos se funciona ou não.

> De qual artigo do estatuto estamos falando? Não encontrei nenhum artigo proibindo isso.

A regra está no documento [Como organizar eventos usando a infraestrutura da APyB](https://docs.google.com/document/d/1j0UwPbZrN1E8W3zdclzyIxaFPgztjLAATSdsPAizFX0/), não no estatuto. @anadulce precisamos corrigir essa parte do texto.

**Reply by alynnefs** [reply]
Talvez nem precise passar de um ano pra outro, se fechar como "pyladies-brasil" ou "pyladies-capítulo-tal" já resolve. Que aí nem atrela a tempo

**Reply by anadulce** [reply]
@rennerocha de fato, queríamos nos referir ao documento citado pelo @rougeth. Já corrigi, obrigada pela observação.
> no documento [Como organizar eventos usando a infraestrutura da APyB](https://docs.google.com/document/d/1j0UwPbZrN1E8W3zdclzyIxaFPgztjLAATSdsPAizFX0/)

**Reply by anadulce** [reply]
> Talvez nem precise passar de um ano pra outro, se fechar como "pyladies-brasil" ou "pyladies-capítulo-tal" já resolve. Que aí nem atrela a tempo

Queremos ter atualizado quem são as pessoas responsáveis pela ponte entre PyLadies e APyB, é esse o motivo dessa "renovação" anual, porque sabemos que há rotatividade entre as pessoas que integram a comunidade.

**Comment by cecivieira** [comment]
@anadulce @tiidadavena @rougeth @muriloviana Aprovamos na última assembleia da Rede PyLadies Brasil as regras para gestão financeira da rede, conforme solicitado pela APyB.

Segue abaixo as regras para conhecimento e demais providências para efetivação do uso desta Associação pela Rede.

[Regras para Gestão Financeira da Rede PyLadies Brasil](https://github.com/pyladies-brazil/organizacao/blob/main/gestao-financeira-rede.md)

**Comment by cecivieira** [comment]
@anadulce @tiidadavena @rougeth @muriloviana oi povo,
temos dinehiro em caixa, no nome de uma pessoa física. Como podemos tranferir para a APyB?
Através de uma doação? Solicitar uma nota fiscal de vocês? Como vamos dar seguimento a essa transferência?

**Comment by cecivieira** [comment]
@jtemporal retomando essa thread.
Estamos prontas para receber o caixa da PyLadies Brasil.
Qual é o valor que será transferido?

Por favor, junto com o valor, envia o link com o fluxo de caixa atualizado no repositório de vocês. Assim, teremos um histórico até a transferência para APyB

**Comment by jtemporal** [comment]
Opa maravilha @cecivieira!

Valor em caixa a ser transferido: **R$ 13.182,59**

O [fluxo de caixa está atualizado para refletir esse valor](https://github.com/pyladies-brazil/organizacao/blob/main/fluxo-caixa.md), aguardo o recebimento dos dados da conta da Apyb para fazer o envio.

**Comment by cecivieira** [comment]
@jtemporal solicitei o recibo/NFe da contadora para formalizar a transação e te enviarei com os dados da nossa conta

**Comment by cecivieira** [comment]
@jtemporal A nota fiscal com os dados bancários para transferência foi enviada para seu e-mail

**Reply by jtemporal** [reply]
Perfeito! Acabei de fazer a transferência do valor via pix.

**Reply by tiidadavena** [reply]
Que notícia boa!

**Reply by tiidadavena** [reply]
@cecivieira a gente pode incluir essa movimentação no roadmap de divulgação?

**Reply by cecivieira** [reply]
@tiidadavena sim
@jtemporal atualiza o [fluxo de caixa](https://github.com/pyladies-brazil/organizacao/blob/main/fluxo-caixa.md) informando sobre a transferência pra apyb. Acho que também tem em algum outro lugar desse mesmo repositório informando que o dinheiro tá na tua conta. Daí já atualiza essa infromação também

**Comment by rougeth** [comment]
@cecivieira vi que foi criado as regras de gestão financeira das Pyladies, e um dos itens no passo a passo do fluxo de pedidos (seção 4) diz "GT Financeiro: Interagir com a APyB para pagamento;". Essa interação já foi documentada no lado da APyB?

**Reply by jtemporal** [reply]
@rougeth até onde me lembro essa interação deverá seguir o mesmo fluxo dos pedidos de pagamento pela APyB hoje, @cecivieira pode me corrigir se eu estiver errada.
